crowdfunding/users/permissions.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import permissions
+
+class IsLoggedIn(permissions.BasePermission):
+    
+    def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+        return obj == request.user
```

Comment:
nice user permission, i would suggest changing the same to "IsCurrentUser" or something a little less generic.
i really like what you've done though!